docs/src/components/ComponentDoc/ComponentProps/ComponentProps.tsx

```diff
@@ -59,8 +59,10 @@ export default class ComponentProps extends React.Component<any, any> {
             <Flex.Item styles={{ display: 'inline-block' }}>
               <>
                 <Input
+                  input={{
```

Comment:
I can't agree with this change because it currently works, we use `partitionHTMLProps` to pass down correct props to `input`.

Reply:
types wouldn't agree with the original implementation, as default element of `Input` is `div` which is rendered

Reply:
should we define `input` as default element type for `Input`? Will do it for now, however, it breaks general Stardust semantics.

`partitionHTMLProps ` seems a serious complication, could we avoid it? What prevents us from rendering `Input` as `input`, with no `div` wrapper? Even from client's perspective it would be the anticipated thing

Reply:
> should we define input as default element type for Input? 

Nope, because default is `div` and it will be used for the `wrapper` slot.

> partitionHTMLProps seems a serious complication, could we avoid it?

This will mean that all attributes should be passed to `input` slot. So we it will be something like this: `<Input input={{ readOnly: true }} />`. At least it looks weird, from other side it means that `readOnly` will be absent in styles and accessibility.

Reply:
if `Input` would be rendered as `input`, why `<Input input='..' />` (i.e. `input` slot) is necessary then?

Reply:
Can you please clarify how [this example](https://stardust-ui.github.io/react/components/input#variations-icon) can be rendered with `<Input as=input />`?

Reply:
there is no way to render this with plain Input, yes, but we could create higher-level `DecoratedInput` component (or the one with other name) which would work for this scenario.

Key thing is that, as a client, I would expect Stardust Component to be equivalent to DOM Element in case if name matches - i.e. I may pass all the same props, expect all the same attributes to provide, etc. Real use-cases and problems from Stardust clients suggest the same expectations, and I might reference issue file for that as well.

This is what drives me towards this thinking - in either case, original issue is now addressed, this should be taken as a separate issue.